docs/ref/django-admin.txt

```diff
@@ -1049,7 +1049,11 @@ Mails the email addresses specified in :setting:`ADMINS` using
 
 .. django-admin:: shell
 
-Starts the Python interactive interpreter.
+Starts the Python interactive interpreter. 
+
+This shell also auto-imports models from your project for you. Additionally, you
+can customize which imports you want or add a new shell runner by following the
+:doc:`custom-shell</howto/custom-shell>` guide.
 
```

Comment:
we would want a `versionchanged` annotation to highlight the new auto-import feature - search other docs for examples